### PR TITLE
unit tests: Collapse, SimpleBox

### LINKED ISSUE #7455: [unit-test] Collapse.vue, SimpleBox.vue
Create unit test for `Collapse`,  `SimpleBox` components.

### PR DESCRIPTION
**Pull request by torchiaf**
Signed-off-by: Francesco Torchia <francesco.torchia@suse.com>

### Summary
Fixes #7455

### CODE REVIEW

File: shell/components/__tests__/Collapse.spec.ts [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+      expect(icon.element).toBeDefined();
+    });
+
+    it('emit', async() => {
+      const collapse = wrapper.find(`[data-testid="collapse-div"]`);
+
+      await collapse.trigger('click');
+
+      expect(wrapper.emitted('update:open')).toBeTruthy();
```
Comment (aalves08) [inline shell/components/__tests__/Collapse.spec.ts:41]: a bit nitpicky but I would also check if the value sent on emit is correct:

`expect(wrapper.emitted('update:open')![0][0]).toBe('whatever-value-to-expect')`

File: shell/components/__tests__/SimpleBox.spec.ts [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    expect(closeButton.element).toBeDefined();
+  });
+
+  it('close emit', async() => {
+    await wrapper.setProps({ canClose: true });
+    const closeButton = wrapper.find(`[data-testid="simple-box-close"]`);
+
+    await closeButton.trigger('click');
+
+    expect(wrapper.emitted('close')).toBeTruthy();
```
Comment (aalves08) [inline shell/components/__tests__/SimpleBox.spec.ts:26]: same as previous comment

**Review by aalves08 — APPROVED** [review]
Left a couple of comments, but good for approval.

I prefer to go a bit further on my unit tests because I like to make sure stuff appears as it should. Ex:

`expect(title.element).toBeDefined();`
I would do:
`expect(title.element).toBe('whatever-value-should-be-here');`

but since we don't have guidelines for unit tests and you've covered the most important parts I think we are good

File: shell/components/__tests__/SimpleBox.spec.ts (hunk 2 above)
Comment (torchiaf) [reply shell/components/__tests__/SimpleBox.spec.ts:26]: Thanks for the review Alex, got you points, it's a good idea to be more specific in verifying expected results.

**Pull request merged**